src/test/java/co/com/bancolombia/task/GenerateDrivenAdapterKotlinTaskTest.java

```diff
@@ -386,7 +387,7 @@ public void generateDrivenAdapterRedisRepositoryForImperative()
             .exists());
   }
 
-  @Test
+  @Test(expected = ValidationException.class)
   public void generateDrivenAdapterRedisRepositoryForReactiveWithSecret()
```

Comment:
should this test generate an exception?

Reply:
Yes, because the reactive repository currently is not compatible with redis.

Reply:
but maybe the test name will be different

Reply:
but maybe the test name will be different